src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -38,6 +62,13 @@ def get(self, request, project):
             if not can_create_tickets and node.id in TICKET_ACTIONS:
                 continue
 
+            # Create as many of these nodes as there are Sentry Apps with UI Alert Rule components
+            if node.id == "sentry.rules.actions.notify_event_sentry_app.NotifyEventSentryAppAction":
```

Comment:
I'd use the same pattern above and put the long ID in a variable in `constants.py` and do 
```python
if node.id in CUSTOM_ACTIONS_ACTIONS:
```

Reply:
Addressed in https://github.com/getsentry/sentry/pull/28659